### PR TITLE
[doc] update trouble shooting

### PR DESCRIPTION
**Pull request by whitewum**
we shall maintain trouble shooting as every RC has its own possible errors.

### CODE REVIEW

File: docs/manual-EN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 1]
```diff
@@ -54,3 +54,40 @@ This is likely caused by the inconsistency between the docker IP and the default
 2. In directory `/usr/local/nebula/etc`, check the config locations of all the IP addresses with the command `grep "172.17.0.2" . -r`.
 3. Change all the IPs you find in step 2 to your container IP 172.17.0.3.
 4. Restart all the services.
+
+## Check logs
+
+They are at `/usr/local/nebula/log/`.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/1.overview/2.quick-start/2.trouble-shooting.md]: Logs are stored under `/usr/local/nebula/logs/` by default.

File: docs/manual-EN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 2]
```diff
@@ -54,3 +54,40 @@ This is likely caused by the inconsistency between the docker IP and the default
...
+
+See [here](docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md)
+
+## Check Configs
+
+They are at `/usr/local/nebula/etc/`.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/1.overview/2.quick-start/2.trouble-shooting.md]: Configuration files are stored under `/usr/local/nebula/etc/` by default.

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 Nebula uses [glog](https://github.com/google/glog) to print logs, gflag to control the severity level of the log, and provides an HTTP interface to dynamically change the log level at runtime to facilitate tracking.
 
+## Log file location
+
+They are at `/usr/local/nebula/logs/`.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: Logs are stored under `/usr/local/nebula/logs/` by default.

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md [hunk 4]
```diff
@@ -29,7 +35,8 @@ Descriptions:
 For example, check the severity minloglevel of storaged:
 
 ```bash
-> curl 127.0.0.1:12000/get_flags | grep minloglevel
+> curl 127.0.0.1:12000/get_flags | grep minloglevel  # storage
+> curl 127.0.0.1:13000/get_flags                     # metad
 ```
 
 Change the logs severity level to **most detailed** with the following command.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: Should we change the log level from 4 to 3 as you did in the Chinese version?
```
> curl "http://127.0.0.1:12000/set_flags?flag=minloglevel&value=3"
```

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md [hunk 5]
```diff
@@ -2,6 +2,12 @@
 
 Nebula uses [glog](https://github.com/google/glog) to print logs, gflag to control the severity level of the log, and provides an HTTP interface to dynamically change the log level at runtime to facilitate tracking.
 
+## Log file location
+
+They are at `/usr/local/nebula/logs/`.
+
+> If you delete the entire log directory (`rm -rf ./*`) at runtime, no log file will output thereafter. You can restart the process to resume log output.
```
Comment (Amber1990Zhang) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: no log file > no logs

Comment (Amber1990Zhang) [inline docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: no log file > no logs

**Comment by whitewum** [comment]
ref #1213

File: docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md (hunk 4 above)
Comment (whitewum) [reply docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: right

File: docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md [hunk 6]
```diff
@@ -53,3 +53,46 @@ nebula> GET CONFIGS graph:load_data_interval_secs
 2. 然后进入配置目录(cd /usr/local/nebula/etc), 查找所有IP地址配置的位置(grep "172.17.0.2" . -r)。
 3. 修改上一步查到的所有IP地址为您的容器地址(172.17.0.3)。
 4. 最后重新启动所有服务。
+
+## Connection refused
```
Comment (wilsonyou) [inline docs/manual-CN/1.overview/2.quick-start/2.trouble-shooting.md]: Refused

File: docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md [hunk 7]
```diff
@@ -2,6 +2,12 @@
 
 Nebula 使用 [glog](https://github.com/google/glog) 来打印日志，使用 gflag 控制日志的级别，并提供 HTTP 接口在运行时动态改变日志级别，以方便追踪问题。
```
Comment (wilsonyou) [inline docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md]: Please change all Nebula to **Nebula Graph**

**Comment by dangleptr** [comment]
Great work.

**Comment by Amber1990Zhang** [comment]
I close this pr and migrant all the changes to #1414.

**Pull request closed without merging**